mappings/net/minecraft/client/gui/screen/ProgressScreen.mapping

```diff
@@ -2,4 +2,10 @@ CLASS net/minecraft/class_435 net/minecraft/client/gui/screen/ProgressScreen
 	FIELD field_2541 title Lnet/minecraft/class_2561;
 	FIELD field_2542 progress I
 	FIELD field_2543 done Z
-	FIELD field_2544 task Lnet/minecraft/class_2561;
+	FIELD field_2544 label Lnet/minecraft/class_2561;
```

Comment:
```suggestion
	FIELD field_2544 stage Lnet/minecraft/class_2561;
```
`label` doesn't give you information so much. Since this field can get updated multiple times in a single task, I would name it `stage`.

Reply:
I've been looking at the GUI side of things too much. Yeah, stage sounds a lot better.